packages/getting-started/src/browser/getting-started-contribution.ts

```diff
@@ -49,11 +57,17 @@ export class GettingStartedContribution extends AbstractViewContribution<Getting
     }
 
     async onStart(app: FrontendApplication): Promise<void> {
-        if (!this.workspaceService.opened) {
-            this.stateService.reachedState('ready').then(
-                () => this.openView({ reveal: true, activate: true })
-            );
-        }
+        this.stateService.reachedState('ready').then(() => {
+            const lastOpenedEditors = this.editorManager.all;
+            if (lastOpenedEditors.length === 0) {
```

Comment:
minor: since the variable is not used more than once we can use it directly:

```suggestion
            if (this.editorManager.all.length === 0) {
```